cms/djangoapps/contentstore/tasks.py

```diff
@@ -58,7 +58,7 @@ def rerun_course(source_course_key_string, destination_course_key_string, user_i
             # it's possible there was an error even before the course module was created
             pass
 
-        return "exception: " + unicode(exc)
+        return ("exception: " + unicode(exc))[:999]  # don't exceed 1000 chars
```

Comment:
Why the 1000 character limit? Is that a SQL thing? Could you use a constant? Maybe it could live in `common/djangoapps/util/db.py`